src/psyclone/domain/gocean/nodes/gocean_extract_node.py

```diff
@@ -88,6 +88,26 @@ def dag_name(self):
         '''
         return "gocean_extract_" + str(self.position)
 
+    # -------------------------------------------------------------------------
+    def update_vars_and_postname(self):
+        '''
+        This function is called after the variables to be extracted
+        have been stored in self._input_list and self._output_list.
+        This functions makes sure that when adding the postfix to
```

Comment:
Suggest:

This function prevents any name clashes that can occur when adding the postfix to output variable names (e.g. if there ...input variable.) We change the ...

I think it would also be nicer if we appended an integer value rather than appending 't's.

Reply:
Yes, I wanted to avoid this, since the loop gets 'somewhat' messier, but you are right (and it's not as messy as I thought).